oshdb-api/src/test/java/org/heigit/bigspatialdata/oshdb/api/tests/TestOSMDataFilters.java

```diff
@@ -139,6 +142,18 @@ public void tagList() throws Exception {
         .timestamps(timestamps1)
         .count();
     assertEquals(5, result.intValue());
+    // only tags
+    result = createMapReducerOSMEntitySnapshot()
+        .osmTag(Arrays.asList(
+            new OSMTag("highway", "residential"),
+            new OSMTag("highway", "unclassified"),
+            new OSMTagKey("building"))
+        )
+        .osmType(OSMType.WAY)
+        .areaOfInterest(bbox)
+        .timestamps(timestamps1)
+        .count();
+    assertEquals(5+42, result.intValue());
```

Comment:
I would propose another test with AND + OR filters mixed like:
```
    // mixed OR + AND filtering
    result = createMapReducerOSMEntitySnapshot()
        .osmTag(Arrays.asList(
            new OSMTag("highway", "residential"),
            new OSMTag("highway", "unclassified"),
            new OSMTagKey("building"))
        )
        .osmTag("name")
        .osmType(OSMType.WAY)
        .areaOfInterest(bbox)
        .timestamps(timestamps1)
        .count();
    assertEquals(?, result.intValue());
```

Reply:
this functionality (that filters can be `AND`-ed together) is already covered by the test below (`tagMultiple`). IMHO it's best to keep tests limited to one specific thing each.